ot/lp/emd_wrap.pyx

```diff
@@ -109,8 +110,10 @@ def emd_c(np.ndarray[double, ndim=1, mode="c"] a, np.ndarray[double, ndim=1, mod
 
     # calling the function
     with nogil:
-        result_code = EMD_wrap(n1, n2, <double*> a.data, <double*> b.data, <double*> M.data, <double*> G.data, <double*> alpha.data, <double*> beta.data, <double*> &cost, max_iter)
-
+        if numThreads == 1:
+            result_code = EMD_wrap(n1, n2, <double*> a.data, <double*> b.data, <double*> M.data, <double*> G.data, <double*> alpha.data, <double*> beta.data, <double*> &cost, max_iter)
+        else:
+            result_code = EMD_wrap_omp(n1, n2, <double*> a.data, <double*> b.data, <double*> M.data, <double*> G.data, <double*> alpha.data, <double*> beta.data, <double*> &cost, max_iter, numThreads)
```

Comment:
what happens if numThreads==2 and omp is not present? Is there a bug or thit is compiled without OMP and just works?

Reply:
If openmp is not supported, then the library is compiled without the omp flag. So every reference to OMP is ignored (pragmas are ignored, and references to omp functions like omp_get_max_threads are replaced). A warning message could be added if necessary, because it is still using the omp code (just not parallelized), which happens to be slower with 1 thread than the older (and not parallelized) version of the solver